crates/weaver_forge/src/filter.rs

```diff
@@ -43,10 +43,22 @@ impl Filter {
             filter: filter_expr.to_owned(),
             error: "No parsed expression".to_owned(),
         })?;
+        let filter = ctx.compile(parsed_expr);
+        if !ctx.errs.is_empty() {
```

Comment:
Strangely, `ParseCtx.compile(...)` doesn't return a `Result` and forces us to check the `errs` field, as was done in the past... I opened an issue in the `jaq` project to improve that in the future.